<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>931</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>771</width>
      <height>601</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </widget>
   <widget class="QWidget" name="leftwidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>131</width>
      <height>600</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QToolButton" name="toolBtnEvent">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>60</width>
       <height>80</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton{
	border-image:url(:/resources/event-unchecked .png);
}
QToolButton:hover
{
	border-image:url(:/resources/event-hover.png);
}
QToolButton:checked
{
	border-image:url(:/resources/event-checked.png);
}

</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>70</width>
       <height>70</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
     <property name="autoRaise">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QToolButton" name="toolBtnCurriculum">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>220</y>
       <width>60</width>
       <height>80</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton{
	border-image:url(:/resources/curriculum-unchecked.png);
}
QToolButton:hover
{
	border-image:url(:/resources/curriculum-hover.png);
}
QToolButton:checked
{
	border-image:url(:/resources/curriculum-checked.png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>70</width>
       <height>70</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QToolButton" name="toolBtnNote">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>60</width>
       <height>80</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton{
	border-image:url(:/resources/note-unchecked .png);
}
QToolButton:hover
{
	border-image:url(:/resources/note-hover .png)
}
QToolButton:checked
{
	border-image:url(:/resources/note-checked .png);
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>70</width>
       <height>70</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>931</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
